pkg/handler/iam_handler.go

```diff
@@ -40,6 +40,9 @@ var (
 	expirationRegex = regexp.MustCompile(`request.time < timestamp\('([^']+)'\)`)
 )
 
+// updateIAMPolicy func that updates the given IAM policy.
+type updateIAMPolicy func(*iampb.Policy, []*v1alpha1.Binding, time.Time) error
```

Comment:
Hi @yolocs , would appreciate your advice on below:
1. Using a func type to reduce duplicative code as much as possible, even though it passes unused parameters. 
2. I remember we want to make the logic of removing expired IAM (line 256) general, but reusability is not the goal in this PR.

Reply:
note: global cleanup